src/beautifier.coffee

```diff
@@ -0,0 +1,2 @@
+Beautifiers = require("./beautifiers")
+module.exports = new Beautifiers()
```

Comment:
What's the reasoning behind creating this file to use as `beautifier` versus just `Beautifier = require('../beautifier')`?

Reply:
An instance of `beautifier` used to be created inside `src/beautify.coffee`, which is the main file in the module. We now require this object to beautify code blocks inside markdown, but we don't want to create circular file references. I moved the creation of the beautifier into this new file and required it inside `src/beautify.coffee` as well as `src/beautifiers/markdown-blocks/index.coffee`.

The existing `src/beautifiers/beautifier.md` exports a class, not an acting object instance.